gc/base/EnvironmentBase.cpp

```diff
@@ -100,8 +100,13 @@ MM_EnvironmentBase::initialize(MM_GCExtensionsBase *extensions)
 	}
 
 #if defined(OMR_GC_MODRON_SCAVENGER) || defined(OMR_GC_VLHGC)
-	if (!_hotFieldStats.initialize(this)) {
-		return false;
+	if(extensions->scavengerTraceHotFields)
+	{
```

Comment:
Here and everywhere below: this bracket should be at the end of previous line. Also please add space after "if"
if (extensions->scavengerTraceHotFields) {

Reply:
👍